app/src/layouts/kanban/kanban.vue

```diff
@@ -165,7 +165,7 @@ function openEditGroup(group: Group) {
 function parseAvatar(file: Record<string, any>) {
 	if (!file || !file.type) return;
 	if (file.type.startsWith('image') === false) return;
-	if (file.type.includes('svg')) return;
+	if (!file.type.includes('svg')) return;
```

Comment:
I'm gonna assume there was a good reason previously to don't render SVGs in here 🤔

Reply:
I believe it was just an oversight. Looking through the commits this code was added in the initial refactor commit by Azri. But if you can confirm @azrikahar that would be great. The Cards layout loads SVGs so I don't know this wouldn't.

Reply:
@ConnorSimply Can we try using a `v-image` for this rather than the `<img>`? This might be one of the reasons of the potential of the app hitting the rate limiter (cc @joselcvarela )